testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
+           originIp: origin.ip
+           source_user: origin.user
+           timestamp: request.time
+           method: request.method | ""
```

Comment:
why does this need a default value (and none of the others do?)

alternatively, could we put `| "-"` on all of the values here and make the template much simpler (removing all the `{or (...) "-"}` bits)?